images/node/Dockerfile.rhel

```diff
@@ -7,6 +7,7 @@ RUN CGO_ENABLED=0 GO_BUILD_FLAGS="-tags no_openssl" make build GO_BUILD_PACKAGES
 FROM registry.svc.ci.openshift.org/ocp/4.2:base
 COPY --from=builder /go/src/github.com/openshift/sdn/openshift-sdn /usr/bin/
 COPY --from=builder /go/src/github.com/openshift/sdn/sdn-cni-plugin /opt/cni/bin/openshift-sdn
+COPY --from=builder /go/src/github.com/openshift/sdn/host-local /opt/cni/container-bin/osdn-host-local
```

Comment:
Bikeshed: put this in `/usr/bin/cni/...`

Reply:
Much cleaner, thanks